cvat/apps/annotation/migrations/0001_initial.py

```diff
@@ -0,0 +1,98 @@
+# Generated by Django 2.1.9 on 2019-07-04 07:19
+
+import cvat.apps.engine.models
+from django.conf import settings
+import django.core.files.storage
+from django.db import migrations, models
+import django.db.models.deletion
+
+import os
+
+def create_builtins(apps, schema_editor):
+    AnnoDumperModel = apps.get_model('annotation', 'AnnoDumper')
+    AnnoParserModel = apps.get_model('annotation', 'AnnoParser')
+
+    path_prefix = ""
+    if "development" == os.environ.get("DJANGO_CONFIGURATION", "development"):
```

Comment:
Why do we need the code at all? Let's keep builtin files in the original place for now.